src/include/libplacebo/utils/libav.h

```diff
@@ -131,12 +131,18 @@ PL_DEPRECATED_IN(v7.343) PL_LIBAV_API void pl_frame_map_avdovi_metadata(
 // values from the `AVDOVIMetadata`.
 //
 // Note: The `pl_dovi_metadata` must be allocated externally.
-// Also, currently the metadata is only used if the `AVDOVIRpuDataHeader`
-// `disable_residual_flag` field is not zero and can be checked before allocating.
+// Currently, the metadata can only be mapped if FEL (full enhancement layer)
+// is not used and this can be checked before allocating
+// (see pl_can_map_avdovi_metadata()).
 PL_LIBAV_API void pl_map_avdovi_metadata(struct pl_color_space *color,
                                          struct pl_color_repr *repr,
                                          struct pl_dovi_metadata *dovi,
                                          const AVDOVIMetadata *metadata);
+
+// Helper function to check if Dolby Vision metadata can be mapped.
+// Returns true if the `AVDOVIRpuDataHeader` `disable_residual_flag` field is
+// not zero or if the NLQ parameters are trivial.
+PL_LIBAV_API bool pl_can_map_avdovi_metadata(const AVDOVIMetadata *metadata);
```

Comment:
`pl_map_avdovi_metadata` could map more metadata, but without `pl_dovi_metadata` use. This could be `pl_is_dovi_metadata_needed` or something and `pl_map_avdovi_metadata` could take NULL if the former function returns false. This would allow to still make RPU values.

Reply:
`pl_map_avdovi_metadata(&color, &repr, NULL, &metadata);` I'm talking about output parameter, when there we decide not to use `pl_dovi_metadata`, but still populate `max_pq` and `avg_pq`. This is how `pl_map_avframe_ex()` works.